### PR TITLE
Helment

### PR DESCRIPTION
**Pull request by sammy-code98**

### CODE REVIEW

File: host/src/components/NotFound/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            ></path>
+          </svg>
+        </div>
+
+        <p className="text-gray-400 mb-6 text-center">The page you're looking for doesn't exist.</p>
+        <Button
+          className="bg-shorttiee_primary text-white font-medium w-full"
+          onClick={handleRedirect}
+        >
+          Button
```
Comment (chimdie) [inline host/src/components/NotFound/index.tsx:43]: Change the button name to the something better

```suggestion
          Return home
```

File: host/src/layouts/Auth.layout.tsx [hunk 2]
```diff
@@ -1,27 +1,63 @@
-import { Link, Outlet } from "react-router-dom";
+import { Link, Outlet, useLocation } from "react-router-dom";
+import HelmetHeader from "@/components/HelmetHeader";
 import { HomeHashtag } from "iconsax-react";
 import { AuthRoutes } from "@/types/routes";
 
+type AuthRoutesPath = (typeof AuthRoutes)[keyof typeof AuthRoutes];
```
Comment (chimdie) [inline host/src/layouts/Auth.layout.tsx:6]: You're duplicating the type `AuthRoutes` here. Use `AuthRoutes` the way you intend to use `AuthRoutesPath`

```suggestion

```

File: host/src/layouts/Auth.layout.tsx [hunk 3]
```diff
@@ -1,27 +1,63 @@
...
+
+const routeHelmetTitle: Record<AuthRoutesPath, { title: string; description: string }> = {
```
Comment (chimdie) [inline host/src/layouts/Auth.layout.tsx:8]:
```suggestion
const routeHelmetTitle: Record<AuthRoutes, { title: string; description: string }> = {
```

File: host/src/layouts/Auth.layout.tsx [hunk 4]
```diff
@@ -1,27 +1,63 @@
...
+  },
+  [AuthRoutes.resetPassword]: {
+    title: "Reset Password",
+    description: "Please enter your new password.",
+  },
+};
+
 export default function AuthLayout(): JSX.Element {
+  const location = useLocation();
+  const currentPath = location.pathname as AuthRoutesPath;
```
Comment (chimdie) [inline host/src/layouts/Auth.layout.tsx:33]:
```suggestion
  const currentPath = location.pathname as AuthRoutes;
```

**Review by chimdie — CHANGES_REQUESTED** [review]

File: host/src/components/NotFound/index.tsx (hunk 1 above)
Comment (sammy-code98) [reply host/src/components/NotFound/index.tsx:43]: ohhh my bad. totally missed that

**Review by chimdie — APPROVED** [review]
LGTM🚀🚀

**Pull request merged**